.github/workflows/cd-sql-engine-populate-persistent-source-schema.yaml

```diff
@@ -0,0 +1,106 @@
+# See [Persistent Source Schema](/GLOSSARY.md#persistent-source-schema)
+# Populating the source schema via this workflow ensures that it's done with the same settings as the tests.
+
+name: Reload Test Data in SQL Engines
+
+# We don't want multiple workflows trying to create the same table.
```

Comment:
Yeah I guess if this ever happens in different schemas the last one in will be wrong anyway.